packages/arb-token-bridge-ui/src/components/TransferPanel/NetworkListbox.tsx

```diff
@@ -33,42 +33,55 @@ export function NetworkListbox({
       value={value}
       onChange={onChange}
     >
-      <Listbox.Button
-        style={{ backgroundColor: color.primary }}
-        className="arb-hover flex w-max items-center space-x-1 rounded px-3 py-2 text-sm text-white md:text-xl"
-      >
-        <span className="max-w-[220px] truncate leading-extra-tight md:max-w-[250px]">
-          {label} {getNetworkName(value.id)}
-        </span>
-        {!disabled && <ChevronDownIcon className="h-4 w-4" />}
-      </Listbox.Button>
+      {({ open }) => (
+        <>
+          <Listbox.Button
+            style={{ backgroundColor: color.primary }}
+            className="arb-hover flex w-max items-center space-x-1 rounded px-3 py-2 text-sm text-white md:text-xl"
+          >
+            <span className="max-w-[220px] truncate leading-extra-tight md:max-w-[250px]">
+              {label} {getNetworkName(value.id)}
+            </span>
+            {!disabled && (
+              <ChevronDownIcon
+                className={twMerge(
+                  'h-4 w-4 transition-transform',
+                  open ? '-rotate-180' : 'rotate-0'
+                )}
+              />
+            )}
+          </Listbox.Button>
 
-      <Listbox.Options className="absolute left-0 z-20 mt-1 max-h-[365px] min-w-full overflow-y-auto overflow-x-hidden rounded border border-white/30 bg-gray-1 font-normal text-white lg:left-auto lg:right-0">
-        {options.map(option => {
-          return (
-            <Listbox.Option
-              key={option.id}
-              value={option}
-              className={twMerge(
-                'hover:white/20 flex h-12 cursor-pointer select-none items-center gap-1 px-3 py-2 transition-[background] duration-200 ui-selected:bg-white/20 ui-active:bg-white/20'
-              )}
-            >
-              <div className="flex h-8 w-8 items-center justify-center">
-                <Image
-                  src={getBridgeUiConfigForChain(option.id).network.logo}
-                  alt={`${getNetworkName(option.id)} logo`}
-                  className="max-h-7 w-auto"
-                  width={35}
-                  height={35}
-                />
-              </div>
-              <span className="max-w-[140px] truncate">
-                {getNetworkName(option.id)}
-              </span>
-            </Listbox.Option>
-          )
-        })}
-      </Listbox.Options>
+          <Transition className="absolute left-0 z-20">
```

Comment:
according to Figma, it was intentional to align it to the right on desktop and to the left on mobile
please add these back `lg:left-auto lg:right-0`

on Figma the "From:" network dropdown should be aligned similarly, i see that it's not on the base branch, i'll fix it on another PR